packages/frontend/src/components/common/ResourceView/index.tsx

```diff
@@ -84,7 +90,11 @@ const ResourceView: React.FC<ResourceViewProps> = ({
         type: ResourceViewItemAction.CLOSE,
     });
 
-    const [activeTabId, setActiveTabId] = useState(tabs?.[0]?.id);
+    const [activeTabId, setActiveTabId] = useState(
+        dashboards?.length === undefined || dashboards?.length === 0
+            ? tabs?.[1]?.id
+            : tabs?.[0]?.id,
+    );
```

Comment:
Hi @CVamsi27, unfortunately we can't fix this bug like this because ResourceView is used in many other cases where the tabs are not 'charts' 'dashboard 'all-items', and we need to be careful not to introduce bugs to other use cases.
Also, the tabs could change in the future and make this obsolete 😅 
It's not an easy bug to fix, we have a draft PR open for it and @IrakliJani is currently investigating it > https://github.com/lightdash/lightdash/pull/5913

Reply:
Possible solution: 
- add new property to set the default active tab

```<ResourceView defaultActiveTab={!dashboards || dashboards.length === 0 ? 'charts' | 'dashboards' }>```

Reply:
Hello @yasmin-e, thanks for that, I thought it's a simple fix as it was tagged with Good First Issue! I was not able to track already implemented PR as the issue was not tagged in the PR.

Hello @ZeRego, should I implement your suggestions in this PR of wait for #5913 
or should I close this PR

Reply:
cc @CVamsi27, @ZeRego 
@IrakliJani just merged a fix for it!
https://github.com/lightdash/lightdash/pull/5938

Reply:
actually @CVamsi27, the PR doesn't fix this bug 😅  I got confused.
You can go ahead and implement the fix Jose suggested on this PR or I can pick it up!
Let me know, and sorry for the confusion.

Reply:
@yasmin-e No worries, I will implement the changes here

Reply:
@yasmin-e Implemented the fix suggested by @ZeRego